chromium_src/components/remote_cocoa/app_shim/native_widget_ns_window_bridge.mm

```diff
@@ -25,4 +27,12 @@
   }
 }
 
+void NativeWidgetNSWindowBridge::ResetWindowControlsPosition() {
+  // Call undocumented method of NSThemeFrame in order to reset window controls'
```

Comment:
more details could be helpful for future us looking at this fragment... Like how did we learn about the undocumented call, what to do if it breaks, etc